api-common/src/main/kotlin/io/qalipsis/api/meters/CampaignMeterRegistry.kt

```diff
@@ -84,6 +84,30 @@ interface CampaignMeterRegistry {
         map: T,
     ): Gauge
 
+    fun summary(
+        scenarioName: ScenarioName = "",
+        stepName: StepName = "",
+        name: String,
+        tags: Map<String, String> = emptyMap(),
+        percentiles: Collection<Double>,
+        histogramCounts: Collection<Double>?,
```

Comment:
We definitely need intensive documentation the function, perhaps with examples? Please also document the other functions.

Reply:
Furthermore, we need default values, in order to avoid specifying everything when they are not needed...

Reply:
I provided the default in the oss. Wouldn't that suffice?

Reply:
The default values for arguments should be part of the interface, not the implementation, like done for the tags on line 91.

Reply:
Okay